arbeitszeit_web/get_statistics.py

```diff
@@ -41,7 +53,60 @@ def present(self, use_case_response: StatisticsResponse) -> GetStatisticsViewMod
                 use_case_response.registered_companies_count
             ),
             registered_members_count=str(use_case_response.registered_members_count),
+            cooperations_count=str(use_case_response.cooperations_count),
+            certificates_count="%(num).2f"
+            % dict(num=use_case_response.certificates_count),
+            available_product="%(num).2f"
+            % dict(num=use_case_response.available_product),
             active_plans_count=str(use_case_response.active_plans_count),
             active_plans_public_count=str(use_case_response.active_plans_public_count),
             average_timeframe_days=average_timeframe,
+            barplot_certificates=self.plotter.create_bar_plot(
+                x_coordinates=[
+                    self.translator.gettext("Work certificates"),
+                    self.translator.gettext("Available product"),
+                ],
+                height_of_bars=[
+                    use_case_response.certificates_count,
+                    use_case_response.available_product,
+                ],
+                colors_of_bars=[self.colors.primary, self.colors.info],
+                fig_size=(5, 4),
+                y_label=self.translator.gettext("Hours"),
+            ),
+            barplot_means_of_production=self.plotter.create_bar_plot(
+                x_coordinates=[
+                    self.translator.gettext("Feste PM"),
```

Comment:
I did not catch this on my last review. But you mixed german translation ids with english translation ids.